pom.xml

```diff
@@ -179,7 +179,7 @@
     <mariadb.version>2.5.0</mariadb.version>
     <postgres.version>42.2.14</postgres.version>
     <opencsv.version>2.3</opencsv.version>
-    <orc.version>1.5.12</orc.version>
+    <orc.version>1.6.7-SNAPSHOT</orc.version>
```

Comment:
So, we cannot use `1.6.6`? What ORC issue was the blocker for this?

Reply:
Hey @dongjoon-hyun -- yeah there are a few issues that are recently resolved but not released as part of 1.6.6, such as: ORC-705 , ORC-706 , ORC-709 and ORC-724

Reply:
Thanks. That's invaluable. Please mention that when you send an email to `dev@orc` mailing list, too.
- https://issues.apache.org/jira/browse/ORC-705
- https://issues.apache.org/jira/browse/ORC-706
- https://issues.apache.org/jira/browse/ORC-709
- https://issues.apache.org/jira/browse/ORC-724